ci/steps.yml

```diff
@@ -217,6 +239,12 @@ steps:
     displayName: Build test with qmake for Linux and macOS w/o extra module
   - powershell: |
       if ( $env:TOOLCHAIN -eq 'MSVC' ) {
+        # Load modules from cache  
+        $Env:PSModulePath = '$(MODULES_FOLDER)', $Env:PSModulePath -join [System.IO.Path]::PathSeparator
+        Write-Host $Env:PSModulePath
+        Import-Module "Pscx"
+        Import-Module "VSSetup"
+
```

Comment:
There is a DRY issue here: This block of code is repeated on lines 275-280. I think it would make a lot more sense to run this code in a previous step, but I do not know how to make the effect of this code persist to future steps. I suspect that there's a way to make this work without repeating this code, but I don't know how.

Reply:
We can define a custom shell  for pipeline that based is pwsh and preloading modules,  but it would be overkill.